pkg/gadgettracermanager/api/gadgettracermanager.proto

```diff
@@ -19,9 +19,6 @@ option go_package = "github.com/inspektor-gadget/inspektor-gadget/pkg/gadgettrac
 package gadgettracermanager;
 
 service GadgetTracerManager {
-  // Methods called via kubectl-exec
-  rpc ReceiveStream(TracerID) returns (stream StreamData) {}
```

Comment:
Which version of kubectl-gadget was using this method?

I guess it is old enough, that we don't need to ensure compatibility between an old client and a new server.

Reply:
We are not using it since we introduced the grpc runtime in https://github.com/inspektor-gadget/inspektor-gadget/pull/1396 (merged in Oct 2023). So, I think we can just remove it.